Fix:

error: parameter 'argc' was not declared, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]

https://github.com/LLNL/yorick/commit/83d52189401e8a07a65203a6f2edafef52fb133e
--- yorick/graph.c.orig
+++ yorick/graph.c
@@ -4095,7 +4095,7 @@ my_rgb_read(Engine *eng, GpColor *rgb, long *nx, long *ny)
 /*--------------------------------------------------------------------------*/
 
 void
-Y_current_mouse(argc)
+Y_current_mouse(int argc)
 {
 #ifdef NO_XLIB
   PushDataBlock(RefNC(&nilDB));
